x-pack/legacy/plugins/maps/common/migrations/add_field_meta_options.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import _ from 'lodash';
+import {  LAYER_TYPE, STYLE_TYPE } from '../constants';
+
+function isVectorLayer(layerDescriptor) {
+  const layerType = _.get(layerDescriptor, 'type');
```

Comment:
remove `_`. `layerDescriptor.type` is always defined. if not's it's a bug, and this will continue to fail anyway.

Reply:
As discussed in person, I would prefer to keep lodash to ensure safe reads. Migrations run when kibana starts and exceptions will prevent kibana server from starting. Migration code should safely handle bad input and not prevent the kibana server from starting.